src/libsyntax/ast.rs

```diff
@@ -627,15 +626,9 @@ pub enum PatKind {
     /// 0 <= position <= subpats.len()
     TupleStruct(Path, Vec<P<Pat>>, Option<usize>),
 
-    /// A path pattern.
+    /// A possibly qualified path pattern.
```

Comment:
The removed comment was no longer relevant and not entirely correct. I wrote some new one instead.